templates/display-sp-tpl.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * This is a simple example of template with table of SPs
+ *
+ * @author Dominik Baránek <0Baranek.dominik0@gmail.com>
+ */
+
+$this->data['header'] = '';
+$this->includeAtTemplateBase('includes/header.php');
+
+$attrNames = $this->data['attrNames'];
+$facilitiesWithAttributes = $this->data['facilitiesWithAttributes'];
+
+$columns = '[';
+$columns .= '{label: "Id", type: "number"},';
+$columns .= '{label: "Name", type: "string"},';
+$columns .= '{label: "Description", type: "string"},';
+if (!empty($attrNames)) {
+	foreach(array_values($facilitiesWithAttributes)[0]['facilityAttributes'] as $attr) {
+		foreach ($attrNames as $attrName) {
+			if ($attrName == $attr->getName()) {
+				$columns .= '{label: "' . $attr->getDisplayName() . '", type: "';
+				if (strpos($attr->getType(), 'Integer')) {
+					$columns .= 'number';
+				} else if (strpos($attr->getType(), 'Boolean')) {
+					$columns .= 'boolean';
+				} else {
+					$columns .= 'string';
+				}
+				$columns .= '"},';
+			}
+		}
+	}
+	$columns = substr($columns, 0, -1) . ']';
+} else {
+	$columns .= ']';
+}
+
+$rows = '[';
+if (!empty($facilitiesWithAttributes)) {
+	foreach ($facilitiesWithAttributes as $facilityWithAttributes) {
+		$rows .= '{c:[';
+		$rows .= '{v: "' . $facilityWithAttributes['facility']->getId() . '"}, ';
+		$rows .= '{v: "' . $facilityWithAttributes['facility']->getName() . '"}, ';
+		$rows .= '{v: "' . $facilityWithAttributes['facility']->getDescription() . '"}, ';
+		foreach ($facilityWithAttributes['facilityAttributes'] as $attribute) {
```

Comment:
Because the $facilityWithAttributes['facilityAttributes'] is map attrName->attr it is better to use somethink like this:
```
foreach ($attrNames as $attrName) {
    $rows .= '{v: "' . $facilityWithAttributes['facilityAttributes'][$attrName]->getValue() . '"}, ';
}

```